style: fix text normal in HowTouchWorks (#18)
KanoComputing/tactile: README.md

Compile libkano_logging and symlink its header files:

> ```git

Reviewer: A bunch of the command blocks in this file aren't `git` and so shouldn't be marked as such. For the most part they are `bash`.

Author: ok as agreed I used bash instead

cd ~/Kano/kano-qt-sdk
sudo mkdir -p /usr/local/include/kano/kano_logging
sudo ln -s `pwd`/lib/kano_logging/includes/* /usr/local/include/kano/kano_logging
qmake
make
sudo ln -s `pwd`/lib/build/* /usr/local/lib
```

### Linux

> ps. This linux README is incomplete (installation issue not solved)

Reviewer: ```suggestion
NB This linux README is incomplete (installation issue not solved)
```

Author: done it

> first update your sistem

Reviewer: ```suggestion
First update your system
```

Author: done it

>       sudo apt-get update

Reviewer: Code block?

Author: done it.

- Install [QtCreator 5.11](https://www.lucidar.me/en/dev-c-cpp/how-to-install-qt-creator-on-ubuntu-18-04/) or higher running these commands:

       sudo apt install build-essential

>        sudo apt install qtcreator

Reviewer: Code block around this guy?

Author: done it

If you want Qt 5 to be the default Qt version to be used when using development binaries like qmake, install the following package:

- `sudo apt install qt5-default`
- `sudo apt install qt5-doc`
- `sudo apt install qt5-doc-html qtbase5-doc-html`

> - `sudo apt install qtbase5-examples`

Reviewer: Group these guys into a big code block

Author: done it

You dont need to set the PATH, linux will do it for you!

> if get the error: *cannot find feature `kano_i18n`*

Reviewer: ```suggestion
If get the error: *cannot find feature `kano_i18n`*
```

Author: done it

> clone the following

Reviewer: Combine this line with the previous - it is a confusing to read like this as it should be one sentence.

Author: done it

repos and have the following structure:

```git
containingFolder
   |-- tactile
   |-- kano-qt-sdk
   |-- kano-toolset
```

commands:

`mkdir containingFolder`

> `mv tactile/ containingFolder/`

Reviewer: Group into one code block

Author: done it

Clone inside containingFolder the repo [kano-qt-sdk](https://github.com/KanoComputing/kano-qt-sdk) and the repo [kano-toolset](https://github.com/KanoComputing/kano-toolset)

```git
Compile libparson and symlink its header files:

cd ~/Kano/kano-toolset/libs/parson
sudo mkdir -p /usr/local/include/parson
sudo ln -s `pwd`/* /usr/local/include/parson
make

```

> in mac we have `libparson.dylib` that in linux is called `libparson.so`

Reviewer: ```suggestion
In mac we have `libparson.dylib` that in linux is called `libparson.so`
```

Author: done it

Author: done it

so replace them.

> so run ```sudo ln -s `pwd`/release/libparson.so /usr/local/lib```

Reviewer: ```suggestion
```bash
sudo ln -s `pwd`/release/libparson.so /usr/local/lib
```
```

Author: done it

```git

> Compile libkano_logging and symlink its header files:

Reviewer: Move this line outside the code block

Author: done it

sudo ln -s `pwd`/lib/build/* /usr/local/lib
```

if you get an error on `make` command
`Project ERROR: Unknown module(s) in QT: qml quick`

You need to install **qml&** and **quick**

run `sudo apt-get install qtdeclarative5-dev`

> ps. you could find `parson.h` in => cd /usr/local/include/parson/lib

Reviewer: ```suggestion
NB You can find `parson.h` in => `/usr/local/include/parson/lib`
```

Author: done it

You could do momentarily a local fix

```git
cd kano-qt-sdk

open => features/kano_build_options.prf

```

modify the file

```git
-macx {
-    LIBS += -L/usr/lib -L/usr/local/lib
-    INCLUDEPATH += /usr/include /usr/local/include
-}
+
+LIBS += -L/usr/lib -L/usr/local/lib
+INCLUDEPATH += /usr/include /usr/local/include
+

```

- run `qmake`

## Alternative way for linux to pass files to the Kano-kit

Enter in kano-kit server and copy the path that you need
You can *find the path* with  the command:

`dpkg-query -L <package name>` => dpkg-query -L tactile

tip: `dpkg-query -L <package name> | grep <name of the file>`

note: this method won't work for a new file, use the install.file instead

or you can **use the install file** in **debian folder** but you must complete manually the path

```git
tactile
   |-- bin
   |-- build
   |-- debian
       |--tactile.install

eg res/ui is inside the kano-kit => /usr/share/tactile
```

example path /usr/share/tactile/

`syntax ssh user@ip` => `ssh laura@172.16.254.180`

you can find your kano-kit ip in the wifi's icon in the kano-kit itself

exit the kano-kit server and run:

> ps. you must use **root** as **user**

Reviewer: There's a lot of sentences beginning with a lower case letter, code blocks incorrectly syntax marked, code blocks encompassing text rather than code and use of `ps` when you probably mean `NB` or `Note` in this file.

Author: I tried to improve it

`rsync -avi <path/where I am going copy from> root@172.16.254.180:/path/Where I going copy to/` => `rsync -av ui root@172.16.254.180:/usr/share/tactile/`

`flag -n` it's gonna tell you the output without running the command for async

**faster approach**

`rsync -avinr res/ui/*  root@172.16.254.180:/usr/share/tactile/ui/`

see if it's ok then run

`rsync -avir res/ui/*  root@172.16.254.180:/usr/share/tactile/ui/`

if you see multiple file being update on your terminal:

```git
<f+++++++++ Tactile/Wires/touch_a_grid.qml
<f+++++++++ Tactile/Wires/touch_a_wire.qml
<f+++++++++ Tactile/Wires/touch_indicator.qml
<f+++++++++ Tactile/Wires/touch_points.qml
<f+++++++++ Tactile/Wires/touchscreen.png
<f+++++++++ Tactile/Wires/wire.png
```

> ## if you change the file you need tp update the debian/changelog file

Reviewer 2: ```suggestion
## If you change the file you need to update the debian/changelog file
```

Author: done it, thank you!

2. You can find the path using the install file in the debian folder
NB You must complete manually the path

```bash
tactile
   |-- bin
   |-- build
   |-- debian
       |--tactile.install

```

Example `res/ui` is inside the kano-kit => `/usr/share/tactile`

Example path `/usr/share/tactile/`

Use this command to login inside the kano-kit's server

Syntax `ssh user@ip` => Example: `ssh laura@172.16.254.180`

**NB** You can find your kano-kit ip in the wifi's icon in the kano-kit itself

Now you can syncronize your local file with the kano kit file

**NB** You must exit the kano-kit server and use **root** as **user** run before to run this command

`rsync -avi <path/where I am going copy from> root@172.16.254.180:/path/Where I going copy to/` => `rsync -av ui root@172.16.254.180:/usr/share/tactile/`

NOTE: `-n` it's gonna tell you the output without running the command for async

#### Faster approach

This command is just going to print the output

`rsync -avinr res/ui/*  root@172.16.254.180:/usr/share/tactile/ui/`

See if it's ok then run the same command without the -n flag

`rsync -avir res/ui/*  root@172.16.254.180:/usr/share/tactile/ui/`

> if you see multiple file being update on your terminal:

Author: ```suggestion
If you see multiple file being update on your terminal:
```

```bash
<f+++++++++ Tactile/Wires/touch_a_grid.qml
<f+++++++++ Tactile/Wires/touch_a_wire.qml
<f+++++++++ Tactile/Wires/touch_indicator.qml
<f+++++++++ Tactile/Wires/touch_points.qml
<f+++++++++ Tactile/Wires/touchscreen.png
<f+++++++++ Tactile/Wires/wire.png
```

> ## if you change the file you need to update the debian/changelog file

Author: ```suggestion
If you change the file you need to update the debian/changelog file
```

> example:

Author: ```suggestion
Example:
```

> if current version tactile (4.2.0-0)

Author: ```suggestion
If current version tactile (4.2.0-0)
```

changelog => tactile (4.3.0-0)

```bash

tactile (4.3.0-0) unstable; urgency=low

  * Change default text formatting to normal

 -- Team Kano <dev@kano.me>  Tue, 12 Mar 2019 15:35:00 +0000

tactile (4.2.0-0) unstable; urgency=low

  * Fixed minor spelling errors
 ...

```

>  if in the changelog file there are already changes with the next version you should just update it.

Author: ```suggestion
 If in the changelog file there are already changes with the next version you should just update it.
```
